nix/shell.nix

```diff
@@ -1,38 +1,54 @@
-# This file is part of the IOGX template and is documented at the link below:
-# https://www.github.com/input-output-hk/iogx#34-nixshellnix
-{ iogxRepoRoot, repoRoot, inputs, inputs', pkgs, system, lib, project ? null, ... }:
+{ repoRoot, inputs, pkgs, lib, system }:
 
 let
-  easyPS = pkgs.callPackage inputs.easyPSSrc { inherit pkgs; };
+
+  purescript = repoRoot.nix.purescript;
+
 in
-{
-  # name = "nix-shell";
-  # prompt = "$ ";
-  welcomeMessage = "Marlowe Runner";
+
+lib.iogx.mkShell {
+
+  name = "marlowe-runner";
+
   packages = [
-    pkgs.podman
     # Please update spago and purescript in `package.json` `scripts` section
-    easyPS."purs-0_15_10"
-    easyPS."purs-tidy"
-    easyPS.purescript-language-server
-    easyPS.pscid
-    easyPS.pulp
-    easyPS.spago
+    purescript.purs-0_15_10
+    purescript.purescript-language-server
+    purescript.purs-backend-es
+    purescript.pscid
+    purescript.pulp
+    purescript.spago
+    purescript.spago2nix
+
+    pkgs.podman
     pkgs.jq
     pkgs.docker
-    pkgs.nodePackages.bower
-    pkgs.nodePackages.jshint
-    pkgs.nodePackages.yarn
-    pkgs.dhall
     pkgs.nodejs-18_x
+    pkgs.nodejs-18_x.pkgs.bower
+    pkgs.nodejs-18_x.pkgs.jshint
+    pkgs.nodejs-18_x.pkgs.yarn
+    pkgs.nodejs-18_x.pkgs.webpack-cli
+    pkgs.dhall
     pkgs.pkg-config
     pkgs.python38
   ];
-  # scripts = { };
-  # env = { };
-  enterShell = ''
-    npm install
```

Comment:
I think that we want to preserve `NODE_OPTIONS` and `PATH` setup in the devel shell. Can we do this?
Side note: Actually whenever we add test suite which can be run on the backend we would like to preserve these options also in the CI as well. Currently this setup is useful for manual testing in the CLI.

Reply:
Sorry, probably `PATH` is not required after nixification - I assume that `npmlock2nix` exposes executables of all the installed JS packages.

Reply:
`npmlock2nix` doesn't expose executables to the shell.
The shell only has the stuff in `nix/shell.nix:packages`.
We can keep `NODE_OPTIONS`. 
I was able to test locally without setting `PATH` too.